backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -89,7 +83,7 @@ export const executeTransaction = async (
     transactionSend.memo = memo
     transactionSend.userId = sender.id
     transactionSend.linkedUserId = recipient.id
-    transactionSend.amount = amount.mul(-1)
+    transactionSend.amount = amount
```

Comment:
You told me to get rid of the mul(-1) on the calls, and implement it as positive.

Anyways, calling it with mul(-1) is not very useful, as it cannot be called with a positive number, it is only used as a sender.

You tell me what to do next, but I changed it because you asked me to do so😅

Reply:
I thought, the we call `calculateBalance` with a positive amount. In our transaction table, the amount of the send transaction must remain negative. This ha nothing to do with the call of `calculateBalance`

Reply:
that is very true, sorry for the misstake
fixed now